infra/dockerfile.base

```
@@ -1,17 +1,17 @@
 FROM python:3.10.7-slim
 
 # Setup dependencies in a cacheable step
-COPY requirements.txt /requirements.txt
-RUN apt-get update && \
-		apt-get install -y --no-install-recommends gcc g++ libgomp1 libffi-dev libjemalloc2 zstd patch && \
-		pip install --disable-pip-version-check --quiet --no-cache-dir -r /requirements.txt && \
-		apt-get purge -y gcc g++ libffi-dev patch && \
-		apt-get autoremove -y && \
-		rm -rf /var/lib/apt/lists/*
+RUN --mount=type=bind,source=requirements.txt,target=/requirements.txt,rw \
```

why do we need `rw`?
```suggestion
RUN --mount=type=bind,source=requirements.txt,target=/requirements.txt\
```

---

Ah right! This isn't relevant here. I'll scrap it 🙂